requirements-dev.txt

```diff
@@ -3,4 +3,4 @@ wheel==0.33.6
 pytest==5.2.1
 twine==3.1.0
 flake8==3.7.9
-black==19.3b0
+black==20.8b1
```

Comment:
because I had a formatting error and did not understand why, found out that it was an outdated version of black so I bumped & reformatted kinda mechanically. Sorry will try to keep PR atomic in the future

Reply:
np